common/app/common/TrailsToShowcase.scala

```diff
@@ -263,9 +263,11 @@ object TrailsToShowcase {
       // Make a questionable inference of the panels publication and update times from it's articles and collection update time
       val updateTimes = articles.map(article => Some(article.updated)) :+ collectionLastUpdated
       val updated = updateTimes.flatten.sortBy(_.getMillis).lastOption.getOrElse(published)
+      // to make the rundown panel ID unique when its title is changed, we'll use a hash of the title as part of the ID
+      val rundownPanelId = id.concat(s"-${panelTitle.replaceAll(" ", "").toLowerCase.hashCode}")
```

Comment:
Is there a reason it starts with `-`?
And should we have the `id` val appended in case there are duplicate container titles?

Reply:
Wait, I've misread the code. It's exactly what I've asked.